polymesh-docs/polymesh-private/confidential-assets/diagrams.md

```diff
@@ -185,11 +184,7 @@ else Venue/Sender does not provide instruction ID
   M ->>P: Requests pending instructions
   P -->> M: Pending instruction ID(s) returned
 end
-M ->> P: Query previously emitted instruction<br> affirmation events to retrieve sender<br>proof(s). A chain indexer (e.g. Polymesh<br> SubQuery) may be required for efficient<br> queries of historical events
-P -->> M: Sender proof
-M ->> P: Query instruction leg details to<br>determine the correct index of<br>the mediators confidential account<br>in the array of auditors
-P -->> M: Auditor index
-M ->> M: Verify or decrypt amount from the sender proof using<br>their confidential account, the confidential asset ID<br>and the auditor index for their confidential account.<br>If known, the expected amount can be<br>provider to reduce the time to verify.
```

Comment:
Revert this change. These steps are still relevant even if it is a case that the SDK or REST API can do them in the background they are still being done.